core/market/decentralized/src/protocol/discovery.rs

```diff
@@ -10,12 +9,11 @@ use ya_core_model::net;
 use ya_core_model::net::local::{
     BindBroadcastError, BroadcastMessage, SendBroadcastMessage, Subscribe, ToEndpoint,
 };
-use ya_service_api_web::middleware::Identity;
 use ya_service_bus::{typed as bus, Handle, RpcEndpoint, RpcMessage};
 
 use super::callbacks::{CallbackHandler, CallbackMessage, HandlerSlot};
 use crate::db::models::{Offer as ModelOffer, SubscriptionId};
-use std::future::Future;
+use crate::matcher::MatcherError;
```

Comment:
Why it's necessary?

Reply:
it is not, but while we did switched off checking `unused_imports` I've missed it